packages/ui/index.tsx

```diff
@@ -204,6 +204,12 @@ export {
 
 export { ScrollArea, ScrollBar } from './src/components/shadcn/ui/scroll-area'
 
+export {
+  ResizablePanelGroup as ResizablePanelGroup_Shadcn_,
```

Comment:
No need for `_shadcn_` suffix since there's no conflict with another component.

Reply:
agreeeed